{"uuid": "7785539c-564f-4eb4-8479-df456f55579f", "children": ["0d98c90a-7799-44ac-9c46-96d40a2c5417"], "befores": [{"name": "driver", "status": "passed", "start": 1691765178256, "stop": 1691765180108}], "afters": [{"name": "driver::0", "status": "broken", "statusDetails": {"message": "AttributeError: 'Function' object has no attribute 'rep_call'\n", "trace": "  File \"C:\\code\\test\\pythonCode\\pythonProject\\ui_pytest_framework\\venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\code\\test\\pythonCode\\pythonProject\\ui_pytest_framework\\venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 917, in _teardown_yield_fixture\n    next(it)\n  File \"C:\\code\\test\\pythonCode\\pythonProject\\ui_pytest_framework\\conftest.py\", line 15, in driver\n    if request.node.rep_call.failed:\n       ^^^^^^^^^^^^^^^^^^^^^\n"}, "start": 1691765183246, "stop": 1691765183246}], "start": 1691765178256, "stop": 1691765183247}